packages/fxa-auth-server/lib/senders/partials/verification_reminder_first.html

```diff
@@ -55,7 +55,7 @@ <h1 style="font-family: sans-serif; font-size: 21px; line-height: 29px; font-wei
   "description": "Reminder: Finish Creating Your Account",
```

Comment:
Does the description need to be translated too?

Reply:
No.